ccd-definition/ComplexTypes/CareSupervision/GatekeepingOrder/StandarDirection.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "StandardDirection",
+    "ListElementCode": "title",
+    "FieldType": "Text",
+    "ElementLabel": "Type",
+    "HintText": "You must include a brief direction summary",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "dueDateType = \"DO NOT SHOW\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "StandardDirection",
+    "ListElementCode": "description",
+    "FieldType": "TextArea",
+    "ElementLabel": "Description",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "showDateOnly = \"NO\""
```

Comment:
because this is based on a hidden field, it will easily break due to how hidden fields are currently handled by XUI. This should hopefully be resolved but we're not sure when, so it can cause some problems (looks like description won't be visible on the CYA page, and if you go back to this page, the description box will be missing)

Reply:
fixed